static/css/marble.css

```diff
@@ -946,4 +872,228 @@ h6{
 
 .registration-status-title{
     font-weight: bold;
+}
+
+/*Media Queries*/
+
+@media (width <= 992px) {
+
+    /*Node Page*/
+    .value-statement-node{
+        width: 85%;
+        text-align: center;
+      }
+
+    .node-menu-header{
+        font-family: var(--secondary-font-family);
+        font-weight: var(--reg-font-weight);
+        font-size: 20px;
```

Comment:
This may look fine on a `sm` size but on `md` it looks really small

Reply:
@ShrutiK100 , can you advise on the font size to use?

Reply:
Made the font size h3 size in screens larger than phone

Reply:
This is which heading exactly?